src/lodash.d.ts

```diff
@@ -0,0 +1 @@
+declare module "lodash";
```

Comment:
This should not be necessary. It's kind of a last resort solution. I believe you can install lodash functions seperately. E.g. Run `npm install lodash/debounce` and install types `npm install @types/lodash`

Reply:
`npm install lodash/debounce` did not work (package not found), `npm install @types/lodash` did work

Reply:
Sorry, it is `npm install --save lodash.debounce`. See https://stackoverflow.com/questions/43479464/how-to-import-a-single-lodash-function

Reply:
Done